js/5etools-importer.js

```diff
@@ -1213,6 +1220,60 @@ function d20plusImporter () {
 		});
 	};
 
+	d20plus.importer._importFilterList = function (importList) {
+		const $winFilterList = $("#d20plus-import-filter-list");
+		$winFilterList.dialog("open");
+		const $btnImport = $winFilterList.find(".btn");
+		const $winText = $winFilterList.find(".table-import-textarea");
+		
+		$btnImport.on("click", () => {
+
+			function filterList () {
+				const toSearch = $winText.val().toLowerCase().replaceAll(`"`, "").split("\n");
```

Comment:
if we have quotes in the input, we should respect them; e.g. `"Namedcreature, Bringer of Fun","QWE"` should be parsed as `name=Namedcreature, Bringer of Fun` `source=QWE`; currently this parses it as `name=Namedcreature` `source=Bringer of Fun`?

some generic CSV parsing ought to do the trick; should be simple enough to put in a utility function for it and call it good